src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+  onTodoClear: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  todo,
+  onTodoClear: todoCleaner,
```

Comment:
The naming convention for your event handler function 'onTodoClear' could be improved. According to the checklist, it's recommended to use the 'handle' prefix for props that represent event handlers. So, 'onTodoClear' could be renamed to 'handleTodoClear'.